website/docs/reference/resource-configs/target_schema.md

```diff
@@ -33,12 +33,13 @@ snapshots:
 </File>
 
 ## Description
-The schema that dbt should build a [snapshot](/docs/build/snapshots) <Term id="table" /> into. Snapshots build into the same `target_schema`, no matter who is running them.
+The schema that dbt should build a [snapshot](/docs/build/snapshots) <Term id="table" /> into. When `target_schema` is provided, snapshots build into the same `target_schema`, no matter who is running them.
 
 On **BigQuery**, this is analogous to a `dataset`.
 
 ## Default
-This is a **required** parameter, no default is provided.
+
+<VersionBlock lastVersion="1.8" >This is a required parameter, no default is provided. </VersionBlock>For versionless dbt Cloud accounts and dbt Core v1.9+, this is not a required parameter. 
```

Comment:
I'm weirdly seeing both of these show up:
<img width="848" alt="Screenshot 2024-09-24 at 4 31 22 PM" src="https://github.com/user-attachments/assets/9b26ee97-13d8-445f-adee-834fbe3f1da5">